src/languages/elixir.js

```diff
@@ -180,12 +180,18 @@ export default function(hljs) {
     beginKeywords: 'defimpl defmodule defprotocol defrecord',
     end: /\bdo\b|$|;/
   });
+  const DEFSTRUCT = {
+    className: 'class',
+    beginKeywords: 'defstruct'
+    // ends immediately
```

Comment:
`class` is deprecated now.  More likely you'd want this to be a newer multi-match rule and then use `title.class` for the name of the struct... See `wren.js` for examples.  I can also do this myself after though... I already had a few commits patching the other declarations to the newer standard but decided they didn't fit in our regex fix PR.

Reply:
Hmm... since the struct's name is not part of the `defstruct` call but it's defined by the module name in which `defstruct` is called, I think I just need to make `defstruct` a keyword instead?

Reply:
```
iex> defmodule User do
...>   defstruct name: "John", age: 27
...> end
```

Ah, yes, just a keyword then for `defstruct`...